discordmenu/embed/text.py

```
@@ -26,6 +29,10 @@ def __init__(self, name: str, value: Union[Box, str]):
             self._value = value
 
     def to_markdown(self) -> str:
+        if not self._name:
```

name should be mandatory, just fail if it's not provided